cmd/client-daemon.go

```diff
@@ -80,7 +85,7 @@ var stopDaemonCmd = &cobra.Command{
 	RunE: func(cmd *cobra.Command, args []string) error {
 		// kill -9 daemon
 		// read from PID file
-		pidFile, err := os.ReadFile("/tmp/cedana.pid")
+		pidFile, err := os.ReadFile("/var/log/cedana.pid")
```

Comment:
`/run/cedana.pid` might be more appropriate location for the pid file

Reply:
good call! fixed